ml-conversational-analytic-tool/test_commentAnalysis.py

```diff
@@ -0,0 +1,53 @@
+import unittest
+from commentAnalysis import CommentAnalyzer
+
+
+class TestCommentAnalysis(unittest.TestCase):
+
+    
+    def test_CommentAnalyzer_init(self):
+        analyzer = CommentAnalyzer(['test'])
+        """ positive test """
+        self.assertEqual(analyzer.word_count, {'test': 0})
+        """ negative test """
+        self.assertNotEqual(analyzer.word_count, {'test': 1})
```

Comment:
I am not a huge fan of having a negative test right after the positive test. Doesn't line 11 imply that test != 1? 

I understand that it's better to be explicit, but this seems a bit too much? WDYT?

Reply:
It's generally better to create a test function per the test you are performing. So one "test_***" to verify the function behaves properly.  Then another "test_<some variation>" to ensure it handles other scenarios. This makes it much easier to observe the result of the test suite and see exactly what failed. If you stuff many asserts into the same function, a developer has to debug the test to know how it failed more granularly.

Reply:
Created separate test functions to have variations. Do let me know what you think about it.